cnf-tests/testsuites/pkg/pods/pods.go

```diff
@@ -76,6 +76,14 @@ func RedefinePodWithNetwork(pod *corev1.Pod, networksSpec string) *corev1.Pod {
 	return pod
 }
 
+// RedefineAsPrivileged updates the pod definition to be privileged
+func RedefineAsPrivileged(pod *corev1.Pod) *corev1.Pod {
```

Comment:
extend this to also have container name and if it's empty it apply to the first container inside the pod